playground/src/import-url/import-url.scss

```diff
@@ -0,0 +1,65 @@
+@import 'mixins.scss';
+@import 'components/atoms/_text.scss';
+
+.import-url {
+  p {
+    font-size: 14px;
+    line-height: 1.64;
+    color: #color('limed-spruce');
+  }
+
+  &-bar {
+    display: flex;
+    height: 40px;
+    margin-top: 30px;
+    border-radius: 4px;
+    box-shadow: 0 10px 25px 0 transparentize(color('black'), 0.8);
+
+    &-input {
+      flex: 1;
+      height: 100%;
+      padding: 0 14px;
+      border: 0;
+      border-radius: 4px 0 0 4px;
+      font-family: Noto Sans;
+      font-size: 16px;
+      font-weight: bold;
+    }
+
+    &-submit {
+      @include txt-font-accent;
+      min-width: 100px;
+      border: 0;
+      border-radius: 0 4px 4px 0;
+      font-family: Poppins;
+      font-size: 16px;
+      text-transform: uppercase;
+      color: color('white');
+      background-color: color('blue-ribbon');
+
+      &.loading {
+        background: color('mercury');
+        @include loading-indicator;
```

Comment:
This being a mixin already prevents code duplication in your source code but would still produce double the CSS. Consider using extends instead.